components/ProductMatrix.vue

```diff
@@ -3,7 +3,7 @@
     <div v-if="error">{{ error }}</div>
     <div class="product-matrix">
       <div v-for="card in cards" :key="card.sortKey">
-        <component :is="components[card.type]" :product="card" />
+        <component :is="components[card.type]" :card="card" />
```

Comment:
Vad är fördelarna med att använda <component :is...>? Istället för <card...>?

Reply:
Vad menar du? Med att använda `<component>`?

Reply:
Ja, koden försvann, konstigt! Jag skrev` <component :is`

Reply:
Hm, jag vet inte haha, det kändes naturligt typ att ha det där så det blir tydligt när man lägger på fler kort i framtiden tror jag

Reply:
Hämtar den automatiskt så att AdCard.vue och InfoCard.vue visas? Förstår inte var kopplingen sker.

Reply:
@coopjakob japp, kolla lite längre ner i denna filen på raderna 35-39